core/chains/evm/forwarders/forwarder.go

```diff
@@ -11,6 +11,8 @@ import (
 type Forwarder struct {
 	ID         int64
 	Address    common.Address
+	EOA        common.Address
+	Dest       common.Address
```

Comment:
These names aren't terrible but in general we don't hate having longer but descriptive column/field names for structs.

Reply:
Yup fair enough